### PR TITLE
VD-42

### PR DESCRIPTION
**Pull request by ahackit**
Updates GeoJsonLayer casing.
Updates Readme

Addresses #42 #43 #44 #45

### CODE REVIEW

**Review by aFraley — APPROVED** [review]
Geo 🍰

File: src/components/layers/index.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 
 import BaseLayerMixin from './BaseLayerMixin'
-import GeoJsonlayer from './GeoJsonlayer'
+import GeoJsonLayer from './GeoJGeoJsonLayersonlayer'
```
Comment (TyrelDenison) [inline src/components/layers/index.js]: is this a gnarly typo?

**Review by TyrelDenison — COMMENTED** [review]
🍰

File: src/components/layers/index.js (hunk 1 above)
Comment (ahackit) [reply src/components/layers/index.js]: Big ol bad copy pasta. Fixed

**Pull request merged**